src/components/reference-book/page.cjsx

```diff
@@ -81,7 +81,7 @@ module.exports = React.createClass
   render: ->
     {courseId} = @context.router.getCurrentParams()
     # read the id from props, or failing that the url
-    cnxId = @getCNXId()
+    cnxId = @getCnxId()
```

Comment:
Aside: I think we should have a quick Discussion about this. Since the letters do not stand for anything I think this is the correct casing of the word `cnx`